apps/www/content/docs/components/spinner.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Spinner
+description: Renders a loading indicator.
+component: true
+---
+
+<ComponentPreview name="spinner-demo" />
+
+## About
+
+The Spinner is built and maintained by [flixlix](https://luca-felix.com).
+
+## Installation
+
+<Tabs defaultValue="cli">
+
+<TabsList>
+  <TabsTrigger value="cli">CLI</TabsTrigger>
+  <TabsTrigger value="manual">Manual</TabsTrigger>
+</TabsList>
+<TabsContent value="cli">
+
+<Steps>
+
+<Step>Run the following command:</Step>
+
+```bash
+npx shadcn-ui@latest add spinner
+```
+
+<Step>Add the `spinner` animation to your tailwind config</Step>
+```js title="tailwind.config.js" {9-12, 16}
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  ...
+  theme: {
+    extend: {
+      ...
+      keyframes: {
+        ...
+        "spinner": {
+          "0%": { opacity: 1 },
+          "to": { opacity: .15 }
```

Comment:
@flixlix  looks like something wrong here

![image](https://github.com/shadcn-ui/ui/assets/142723369/7ae938e5-9100-47ba-90c9-caa572073fa4)

and can you double check this 

```json
==> "0%": 
==> "to":
```

I think I should be 

```json
spinner: {
  from: { opacity: 1 },
  to: { opacity: 0.15 },
},
```

but still I see the TS error 
`Type 'number' is not assignable to type 'string'.ts(2322)`

this is just a TS error the component works perfectly fine.

![image](https://github.com/shadcn-ui/ui/assets/142723369/c829cf8b-8388-4b26-87d8-27b26abe5dd6)